From foo@baz Tue Apr  9 12:12:43 2002
Date: Fri, 08 Jun 2007 12:08:34 +0800
To: Greg KH <greg@kroah.com>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: PCI: remove cpqphp driver maintainer

I'm no longer maintaining this driver as I no longer have the hardware
and I doubt anyone else does either.


Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 MAINTAINERS |    5 -----
 1 file changed, 5 deletions(-)

--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -2814,11 +2814,6 @@ P: 	Kristen Carlson Accardi
 M:	kristen.c.accardi@intel.com
 S:	Supported
 
-PCI HOTPLUG COMPAQ DRIVER
-P:	Greg Kroah-Hartman
-M:	greg@kroah.com
-S:	Maintained
-
 PCIE HOTPLUG DRIVER
 P:	Kristen Carlson Accardi
 M:	kristen.c.accardi@intel.com
